.mergify.yml

```diff
@@ -15,8 +15,11 @@ pull_request_rules:
         type: APPROVE
         message: MickeyMoz 💪
       merge:
-        method: rebase
-        strict: smart
+        # These PRs are just bumping versions, and very unlikely to break
+        # anything. It's not worth the delay of rebasing them
+        # (sometimes multiple times) before merging.
+        method: squash
```

Comment:
nit: Wondering why we switch to `squash` here? Couldn't this be `rebase` with `strict: false`? I guess it doesn't matter much for the "single commit" PRs that the bot currently opens.

But apart from that, I just checked the ([docs](https://docs.mergify.io/actions/merge/) and it looks like the merge options are deprecated and have now more fine-grained queue controls (although only for premium plans?).

Reply:
Yeah, that's a good point. When I wrote this I thought that we couldn't rebase without being up-to-date, but that's silly in retrospect. I'll revert this part.